### PR TITLE
Improve UG find command

### PR DESCRIPTION
**Pull request by ricketytoc**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -147,36 +147,92 @@ Examples:
...
-* For finding by Name, Email, Department and Role, **matching at least one keyword** will be returned.
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang` e.g. `alex yeoh` will return `alextan@example.com`, `berniceyeoh@example.com`.
-* For finding by Salary, command will find employees whose salaries fall within the range of LOWERBOUND to UPPERBOUND inclusive of LOWERBOUND and UPPERBOUND.
-* The LOWERBOUND **cannot be larger than** the UPPERBOUND. 
-* There must be a **spacing before and after the dash**
-* LOWERBOUND and UPPERBOUND must be **non-negative integers** and **cannot be larger than 1 000 000 000**. 
+* Finds employees whose attributes satisfy all the requirements of the FindCommand
+
+Examples:
+* `find i/A00 n/alex p/9001` returns employees whose ID contains `A00`, whose name contains the word `alex` and whose phone number contains `9001`.
```
Comment (Weiennn) [inline docs/UserGuide.md:161]: Is it better to just use one "whose"? so
whose ID contains `A00`, name contains the word `alex` and phone number contains `9001`.

**Review by Weiennn — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -147,36 +147,92 @@ Examples:
...
+Examples:
 * `find e/alex bernice` returns employees with emails `alexyeoh@example.com`, `bernicetan@example.com`.
+
+#### Finding an employee by ID: `find i/`
+
+Finds employees whose ID contains the keyword.
+
+Format: `find i/KEYWORD`
+
+* The command will find employees whose ID and Phone number contains the respective keywords.
```
Comment (Goh-Li-Ting) [inline docs/UserGuide.md:212]: Is this supposed to be just ID, and not "ID and Phone number"? (and also "respective")

File: docs/UserGuide.md (hunk 1 above)
Comment (ricketytoc) [reply docs/UserGuide.md:161]: Ok!

File: docs/UserGuide.md (hunk 2 above)
Comment (ricketytoc) [reply docs/UserGuide.md:212]: Oh yes you're right! Thanks so much for spotting this mistake

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

**Pull request merged**